src/services/targetService.js

```diff
@@ -12,6 +12,10 @@ class TargetService {
   createTarget(target) {
     return api.post('/targets', target);
   }
+
+  deleteTarget(target) {
+    return api.delete(`/targets/${target}`);
```

Comment:
Will be better to use the name `targetId ` instead of `target`, because target seems to be an object but is the id number of the target